doc/api/util.md

```diff
@@ -325,6 +325,60 @@ The `--throw-deprecation` command-line flag and `process.throwDeprecation`
 property take precedence over `--trace-deprecation` and
 `process.traceDeprecation`.
 
+## `util.diff(actual, expected)`
+
+<!-- YAML
+added: REPLACEME
+-->
+
+* `actual` {any}
+* `expected` {any}
+* Returns: {string | null} A string if there is difference
```

Comment:
Nit: should we just always return a string? It being empty in case there is no difference?